### ISSUE TITLE
Training set == validation set ?

### ISSUE THREAD

**Issue by fulcus** [XLearning-SCU/2022-IJCV-TCL#2]
Hi, thank you for releasing the code of your model. 
My question is, why is the model trained and validated using the same data? My understanding is that this invalidates the claimed performance, which is supposed to be on an untouched test set. 
https://github.com/XLearning-SCU/2022-IJCV-TCL/blob/9248f727986e5db7582500e227bfb26970a2f9ee/train.py#L135
https://github.com/XLearning-SCU/2022-IJCV-TCL/blob/9248f727986e5db7582500e227bfb26970a2f9ee/data.py#L154-L170

Please correct me if I'm wrong, thanks!

**Comment by fulcus** [comment]
Ok, I just realized my comment doesn't make sense. The method is fully unsupervised, so the acc is calculated on all the data.
